deploy/charts/csi-driver-spiffe/values.yaml

```diff
@@ -218,3 +219,45 @@ priorityClassName: ""
 
 # Labels to apply to all resources
 commonLabels: {}
+
+# Kubernetes node selector: node labels for pod assignment.
+# +docs:property=nodeSelector
+nodeSelector:
+  kubernetes.io/os: linux
```

Comment:
I wondered if this would cause issues for existing installations... It seems like we only have linux images (amd64 and arm64), so I assume that this is fine. 🤞

Reply:
It seems like we do not set this default in other Helm charts (see eg. https://github.com/cert-manager/csi-driver/blob/main/deploy/charts/csi-driver/values.yaml#L127-L131)

Reply:
Is there a chance it will break anyone? 

Our best practice page is telling people to set nodeSelector to linux: https://cert-manager.io/docs/installation/best-practice/, so why not set it here by default? Wouldn’t it make sense?

Reply:
I would start with leaving it empty.
If we want to set the default value, we should go through all our Helm charts and set this value everywhere.

Reply:
OK, I'll do that

Reply:
We do set this for cert-manager: https://github.com/cert-manager/cert-manager/blob/1035bb44ca8e9a1cd49c9de79dd5b28af1dcc972/deploy/charts/cert-manager/values.yaml#L347

I see no reason to not set it here

Reply:
Thanks @SgtCoDFish for spotting that! I've reverted the change in 36e6da5404a269151515951310dc1317235c1c2b, csi-driver-spiffe now defaults to the recommended `nodeSelector: {kubernetes.io/os: linux}`.